deployment/ansible/host_vars/13.250.162.250/vars.yml

```diff
@@ -0,0 +1,2 @@
+public_peer_key: files/peer_keys/{{ public_ip }}.pub
```

Comment:
hmm, looking on this it's a lot of duplication, let's rework it:

create new group in a static inventory file `inventory/static.ini`:
```
[uat_seed_nodes]
52.10.46.160
18.195.109.60
13.250.162.250
31.13.249.70
```

Then create `group_vars/uat_seed_nodes/vars.yml` with the content of one of those 4 (duplicating) files.

Reply:
ok